spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/SparkWriteConf.java

```diff
@@ -721,9 +725,4 @@ public DeleteGranularity deleteGranularity() {
         .defaultValue(DeleteGranularity.FILE)
         .parse();
   }
-
-  public boolean useDVs() {
-    TableOperations ops = ((HasTableOperations) table).operations();
-    return ops.current().formatVersion() >= 3;
-  }
```

Comment:
I ended up removing this since I realized it's not an officially released public API on SparkWriteConf. We can just update deleteFileFormat() to return PUFFIN and then use that in SparkPositionDeltaWrite

Reply:
SGTM, I had a similar concern here: https://github.com/apache/iceberg/pull/11588/files#r1916175700